<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="login_SW">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>831</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;메인 페이지&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>306</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_Button">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>420</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>304</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>아이디 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_Button">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>360</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id_lineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pw_lineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>250</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_3">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>265</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>205</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_6">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>306</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>|</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>115</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;로그인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>511</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Home1_Button">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <zorder>signup_Button</zorder>
    <zorder>sh_label_4</zorder>
    <zorder>login_Button</zorder>
    <zorder>login_id_lineEdit</zorder>
    <zorder>login_pw_lineEdit</zorder>
    <zorder>sh_label_3</zorder>
    <zorder>sh_label_2</zorder>
    <zorder>sh_label_6</zorder>
    <zorder>sh_label</zorder>
    <zorder>line</zorder>
    <zorder>Home1_Button</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="sh_label_7">
     <property name="geometry">
      <rect>
       <x>456</x>
       <y>41</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;GyeonggiBatang&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;회원가입&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>81</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_8">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>181</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;아이디&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_9">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>231</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_10">
     <property name="geometry">
      <rect>
       <x>309</x>
       <y>281</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호 확인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_11">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>334</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;연락처&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_12">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>381</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;주소&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_13">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>131</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;이름&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>171</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>221</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw2_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>271</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>321</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>숫자만 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>371</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_lineEdit">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>111</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="duplication_Button">
     <property name="geometry">
      <rect>
       <x>696</x>
       <y>173</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>421</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_14">
     <property name="geometry">
      <rect>
       <x>306</x>
       <y>471</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;이용약관동의&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agree1_checkBox">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>471</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>전체 동의 합니다.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agree2_checkBox">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>501</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>이용약관 동의</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agree3_checkBox">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>531</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>개인정보 수집, 이용 동의(필수)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="agree4_checkBox">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>561</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>개인정보 수집, 이용 동의(선택)</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_15">
     <property name="geometry">
      <rect>
       <x>676</x>
       <y>511</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>약관 보기&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_16">
     <property name="geometry">
      <rect>
       <x>676</x>
       <y>571</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>약관 보기&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_17">
     <property name="geometry">
      <rect>
       <x>676</x>
       <y>541</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>약관 보기&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_Button">
     <property name="geometry">
      <rect>
       <x>566</x>
       <y>621</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Home2_Button">
     <property name="geometry">
      <rect>
       <x>346</x>
       <y>621</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
